src/main/java/org/broadinstitute/hellbender/engine/filters/MarkDuplicatesReadFilter.java

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.spark.transforms.markduplicates.MarkDuplicatesSparkUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+
+/**
+ * Filters out reads marked as duplicates.
+ */
+public final class MarkDuplicatesReadFilter extends ReadFilter implements Serializable {
```

Comment:
Wasn't aware of NotDuplicateReadFilter. Changed the class to NotOpticalDuplicateReadFilter.